src/store/modules/app.js

```diff
@@ -316,7 +410,7 @@ const mutations = {
     state.tutorialEnabled = !state.tutorialEnabled;
   },
   setDarkmode(state, enabled) {
-    state.darkmodeEnabled = enabled;
+    state.configuratorSettings.darkmodeEnabled = enabled;
```

Comment:
configurator settings is starting to smell, I wonder if it just needs to be it's own module